pkg/config/setup/config.go

```diff
@@ -691,6 +691,25 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.SetKnown("snmp_listener.ping.linux")
 	config.SetKnown("snmp_listener.ping.linux.use_raw_socket")
 
+	config.SetKnown("network_devices.snmp_listener.discovery_interval")
```

Comment:
**💭 thought:** ‏it might be worth to comment that `network_devices.snmp_listener` is preferred / has precedence over `snmp_listener` and that the config is here for legacy reason imho

Reply:
yes good idea
https://github.com/DataDog/datadog-agent/pull/24644/commits/07cc321ec7ef6c6029ba5e0b374d0d1b9c1703ec